sdk/client.py

```diff
@@ -49,7 +124,7 @@ def _login(self) -> None:
         }
 
         rsp = requests.post(
-            url,
+            self.login_url,
```

Comment:
We build a URL above but do not use it ?

Reply:
Sorry, remnant from earlier version.